### PR TITLE
Initial paypal implementation

### PR DESCRIPTION
**Pull request by ifyio**

### CODE REVIEW

File: client/routes/signup-form/controller.js [hunk 1]
```diff
@@ -60,6 +62,30 @@ export default class SignupForm {
 			});
 		}
 
+//		view.onCLickingThePaypalBtn(()=> {
+//			dispatch(initiatePaypalFlow)
+//		});
```
Comment (lc512k) [inline client/routes/signup-form/controller.js]: should this be removed?

Comment (ifyio) [inline client/routes/signup-form/controller.js]: yes it should. Will be removed in cleanup

File: views/form-poster.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<title></title>
+	</head>
+	<body>
+
+		<script>
+			var fields = JSON.parse('{{{fields}}}');
```
Comment (lc512k) [inline views/form-poster.html]: Could we move this script to a JS file?

Comment (ifyio) [inline views/form-poster.html]: That page is there to facilitate posting to another page via server-side code. Currently it's not possible to do that server-side so this page was created to act as an intermediary. To post to a page, we simply redirect to this page with the form fields as the query param. Then this page will do a form post to the target page. It's how the paypal code is posting to the `/thank-you` page.

**Pull request merged**